### ISSUE TITLE
No output from pod setup

### ISSUE THREAD

**Issue by AntSwift** [CocoaPods/CocoaPods#9184]
# Report

## What did you do?
On Mac OSX Mojave (clean install with Android Studio and Xcode added) I want to run `pod setup`.
I first attempted to install CocoPods using `sudo gem install cocoapods` then attempted using `sudo gem install -n /usr/local/bin cocoapods`.

## What did you expect to happen?
The  CocoPods environment should be downloaded and set up.

## What happened instead?
It seems nothing, early exit, no output in terminal.

```
My-MBP:~ me$ pod setup
My-MBP:~ me$ 
```

### Stack

```
   CocoaPods : 1.8.0
        Ruby : ruby 2.3.7p456 (2018-03-28 revision 63024) [universal.x86_64-darwin18]
    RubyGems : 2.5.2.3
        Host : Mac OS X 10.14.6 (18G95)
       Xcode : 11.0 (11A420a)
         Git : git version 2.21.0 (Apple Git-122)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib
Repositories : master - git - https://github.com/CocoaPods/Specs.git @ 09c9973be46f66393f762a7c611c2f53fad4eb55
```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.4
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.1.0
cocoapods-trunk       : 1.4.0
cocoapods-try         : 1.1.0
```

**Comment by dnkoutso** [comment]
Yes that is intended. The current 1.8.0 `pod setup` does not do anything and we have kept it if we need it in the future and to avoid breaking systems that depend on it.

I can keep this open with a tiny enhancement that prints out a message the "Setup completed successfully." or something.

**Comment by AntSwift** [comment]
Thanks @dnkoutso! Agree that some output would be helpful, thanks.

**Comment by acefeng** [comment]
Are there any other options besides pod setup?

**Comment by harrisbaitalrayan** [comment]
how can i know pod setup is complete ? pod setup shows no message and finishes quickly, but "flutter doctor" still says pod not setup. This is even after a long time.

**Comment by AntSwift** [comment]
> how can i know pod setup is complete ? pod setup shows no message and finishes quickly, but "flutter doctor" still says pod not setup. This is even after a long time.

I suspect you are using the latest version of cocoapods, try downgrading to 1.7.5.

**Comment by dnkoutso** [comment]
that message probably comes from flutter, not from the `pod setup` command but I am not 100% certain of that.

**Comment by dnkoutso** [comment]
PR merged and will ship with whenever 1.9.0 ships.
